requirements.txt

```diff
@@ -3,10 +3,11 @@ flask-caching
 PyYAML>=3.11
 redis>=2.10.5
 rq>=0.6.0
-compliance-checker>=4.3.2
+-e git://github.com/ioos/compliance-checker.git#egg=compliance-checker
```

Comment:
@daltonkell sorry this slipped past me. This repo should only use stable releases of the compliance checker